### PR TITLE
feat: add Table HOC with group actions

### PR DESCRIPTION
**Pull request by krozhkov**
Add a new table HOC with group actions. Based on discussion in UXRFC-200.

### CODE REVIEW

File: src/components/TableHOCs/withTableSelectionAndGroupActions/withTableSelectionAndGroupActions.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import type {TableDataItem, TableProps, WithTableSelectionProps} from '@gravity-ui/uikit';
+
+import {ActionsPanel, ActionsPanelProps} from '../../ActionsPanel';
+import {block} from '../../utils/cn';
+
+import './withTableSelectionAndGroupActions.scss';
+
+const b = block('with-table-group-actions');
+
+type ActionItem = ActionsPanelProps['actions'][number];
```
Comment (korvin89) [inline src/components/TableHOCs/withTableSelectionAndGroupActions/withTableSelectionAndGroupActions.tsx:14]: U can get this type [here](https://github.com/gravity-ui/components/blob/bc3514b556152a2c5c8dca0a644d8dba69694d50/src/components/ActionsPanel/types.ts#L3)

File: src/components/TableHOCs/withTableSelectionAndGroupActions/withTableSelectionAndGroupActions.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        const selected = props.selectedIds.length;
+        const renderTitle = useMemo(() => () => formatTitle(selected), [formatTitle, selected]);
+
+        return (
+            <>
+                <TableComponentWithSelection
+                    {...(props as TableProps<I> & WithTableSelectionProps<I> & E)}
+                />
+                {selected > 0 && (
+                    <ActionsPanel
```
Comment (korvin89) [inline src/components/TableHOCs/withTableSelectionAndGroupActions/withTableSelectionAndGroupActions.tsx:51]: Is it okay that `ActionsPanel` will be required specific markup of it parent? For example - parent with flex and row direction will corrupt componet's view

**Review by korvin89 — COMMENTED** [review]
I think we can add some basic test as well

File: src/components/TableHOCs/withTableSelectionAndGroupActions/withTableSelectionAndGroupActions.tsx (hunk 2 above)
Comment (krozhkov) [reply src/components/TableHOCs/withTableSelectionAndGroupActions/withTableSelectionAndGroupActions.tsx:51]: But how can we avoid this gracefully? Use portals? I don't see a perfect solution.

**Pull request closed without merging**